Poeschl/glances
# Versioned minimal full images

Test request

Notes:

* Dockerfile Rewrite to multistaged build
  * Added full image variant
  * latest size diff -> -138MB
  * with layers better caching possible

* Actions rewrite
  * pipy release first, to ensure getting the latest package on docker
  * Matrix-build for docker images
    * Build images in parallel

* Add alpine docker images for default and full images
